src/modules/tables/tables.controller.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Put,
+  Param,
+  Delete,
+  UseGuards,
+} from '@nestjs/common';
+import { TablesService } from './tables.service';
+import { CreateTableDto } from './dto/create-table.dto';
+import { UpdateTableDto } from './dto/update-table.dto';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { AuthGuard } from '@nestjs/passport';
+
+@ApiTags('The tables')
+@Controller('tables')
+export class TablesController {
```

Comment:
Why is the module name  plural and not singular like the rest?

Reply:
You are right i'm going to rename it.